jwt_lib.cpp

```diff
@@ -0,0 +1,119 @@
+#include <M5StickC.h>
+#include <Arduino.h>
+#include "mbedtls/base64.h"
+#include "mbedtls/sha256.h"
+#include "mbedtls/pk.h"
+#include "mbedtls/ecdsa.h"
+#include "jwt_lib.h"
+
+int Bearer_JWT_Token(const char* private_key, char* kid, char* serial_num, int time_iat, int time_exp,char* bearer_token)
+{
+    int err_ret = 0;
+    char buffer[600];
+    size_t h64_len;
+    size_t c64_len;
+    size_t s64_len;
+    size_t index;                                                                                         //JWT Token variables
```

Comment:
Tie kommenty sú mimo, daj ich niekam, kde ich je vidno, takto keď ich vidíš, a to je veľké keď, záležiace od toho, čo za editor používaš, a aké veľké okná/monitor máš, tak nevieš, ku ktorému riadku patria ibaže začneš lúštiť